### DISCUSSION TITLE
Failed to download file from Qfieldcloud

### DISCUSSION THREAD

**Discussion by juwizeye** [opengisch/QField#2551] [Q&A]
Hi, 
 I try to download files from Qfieldcloud during sync, but it doesn't work. The files get uploaded up to 100%, but nothing happens.

![image](https://user-images.githubusercontent.com/96420889/156213139-1ac95300-cf3f-4f84-8170-dce014aa8aa1.png)

**Comment by suricactus** [comment]
Please share your QGIS and QFieldSync versions. I guess if you upgrade to QFieldSync 4.0.0 BETA20 it will solve your issues.
